### DISCUSSION TITLE
Volunteer out-reach

### DISCUSSION THREAD

**Discussion by daniel-vnext** [alan-turing-institute/AutSPACEs#372] [Ideas]
# Overview

*As a person affected by autism
I want to be made aware of the project
So that I can contribute and enrich the project*

## Idea

There's a question over how to encourage community participation. A couple of ideas are as follows:

* out-reach for autistic schools / learning support, to see if younger people want to participate
* out-reach for support workers

**Comment by GeorgiaHCA** [comment]
This user story is based on recommendations from an Autistic commnuity member given at an open call to showcase work on the project.

The original notes fom that meeting which this user story is taken and adapted from are here: https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/workshops/developer-showcase-August2020.md

**Comment by GeorgiaHCA** [comment]
This is definitely worth ongoing consideration. This issue should remain open as it will be a constant feature of the projcet. Anyone is welcome to add more ideas here or raise them as issues!

**Comment by lottycoupat** [comment]
Is this very similar to issue alan-turing-institute/AutSPACEs#299 @GeorgiaHCA - or do we want to keep a record of these user stories as issues?
